curriculum/challenges/english/20-upcoming-python/learn-lambda-functions-by-building-a-expense-tracker/657eca6e8965365ab6e57a79.md

```diff
@@ -0,0 +1,79 @@
+---
+id: 657eca6e8965365ab6e57a79
+title: Step 22
+challengeType: 20
+dashedName: step-22
+---
+
+# --description--
+
+In the elif block for choice '3', we'll call the `total_expenses` function to calculate the sum of all expenses and print the result.
+
+Now inside your elif statement, print the total expenses by calling the `total_expenses` function with `expenses` as its argument and using the `print()` function to print the result. A sample syntax for this is:
+
+```py
+print("\nTotal Employees: ", total_employees(employees))
```

Comment:
```suggestion
print('\nTotal Employees: ', total_employees(employees))
```

Reply:
For consistency with the other projects, replacing double quotes by single quotes.